plugin-guide.md

```diff
@@ -64,10 +96,20 @@ Output provisioners implement `ca.on.oicr.gsi.vidarr.OutputProvisionerProvider`
 and `ca.on.oicr.gsi.vidarr.OutputProvisioner`. These plugins are responsible
 for taking data (files or JSON values) from completed workflows, moving the
 data into permanent storage and writing back a file path or URL that will be
-associated with the correct external identifiers.
+associated with the correct external identifiers. Output provisioners can
+choose to handle only some kinds of output data (files, logs, data-warehouse
+entries, or QC judgements) and the system administrator can choose multiple
+provisioners to handle all the input types they require.
+
+Output provisioners are run twice for each workflow: a preflight and a
+provision out. The preflight is run before the workflow has started and allows
+the plugin to validate any configuration metadata provided by the submitter to
```

Comment:
'the submitter' is a completed workflow, right? this wording sounds a little like that workflow is explicitly pushing data to the OutputProvisioner and I can't tell whether that's accurate or not

Reply:
The submitter is Shesmu/the olive. The output provision gets data from the olive (the `metadata`) and data from the output of the workflow. The first pass checks the data from the olive and the second gets that same data supplemented with the output from the workflow.

I'm not sure how to explain it.

Reply:
It sounds like you just did, i was just confused by the wording